justfile

```diff
@@ -36,6 +36,14 @@ frb-generate $CARGO_TARGET_DIR=(justfile_directory() + "/target/frb_codegen"):
     rm -Rf lib/core/api lib/core/frb_*.dart lib/core/lib.dart
     flutter_rust_bridge_codegen generate
 
+# compare the generated files with the current files
+frb-compare $CARGO_TARGET_DIR=(justfile_directory() + "/target/frb_codegen"):
+    rm -Rf /tmp/frb-temp-files
+    cp -R . /tmp/frb-temp-files
+    (cd /tmp/frb-temp-files/app && flutter_rust_bridge_codegen generate --dart-output /tmp/frb-temp-files/app/lib/core)
+    (cd /tmp/frb-temp-files/app && dart run build_runner build --delete-conflicting-outputs)
+    diff -r /tmp/frb-temp-files/app/lib/core app/lib/core
```

Comment:
Wouldn't it be better to generate the files inside the git repository and use git status --porcelain?

```bash
if [[ -n "$(git status --porcelain)" ]]; then
  echo "Repository is dirty! Did you forget to run `just frb-generate`?"
  git diff
  exit 1
fi
```

I think the git approach is more robust, because it will ignore all ignored files but catch any file we might have missed in this comparison.

Reply:
Done in just as proposed. Tested in CI.